extensions/vscode/README.md

```diff
@@ -254,3 +254,51 @@ Finally you need to make VS Code recognize your new extension and automatically
 - [angular](https://github.com/angular/angular) shows how TS server plugin working with language service.
 - Syntax highlight is rewritten base on [vue-syntax-highlight](https://github.com/vuejs/vue-syntax-highlight).
 - [vscode-fenced-code-block-grammar-injection-example](https://github.com/mjbvz/vscode-fenced-code-block-grammar-injection-example) shows how to inject vue syntax highlight to markdown.
+
```

Comment:
The following content is automatically generated via [`vscode-ext-gen`](https://github.com/antfu/vscode-ext-gen)